app/src/main/java/cto/its/ny/gov/fakenews/MainActivity.kt

```diff
@@ -23,110 +24,36 @@ class MainActivity : AppCompatActivity() {
         setContentView(view)
 
         val amountOfArticles = 0..5
-        createArticles(amountOfArticles)
+        val nSvc = NewsService()
```

Comment:
Just a suggestion, but this could get moved out of the `onCreate()` method and marked as a `private val`.  This way it could be referenced outside of just this method in the future.